tasks/sst_metrics_utils/separations.py

```diff
@@ -0,0 +1,228 @@
+import numpy as np
```

Comment:
I'm still a little uncomfortable with copying code out of `validate_drp` and not just calling it as a library.  I don't see any reason not to have a dependency on `validate_drp`.

Reply:
I was envisioning `validate_drp` being deprecated at some point (though probably no time soon), at which point we will want to extract many of its utilities. That can wait, though. I'll remove the duplicated code and just leave the new function in `separations.py`.